apis/core/v1beta1/openstackcontrolplane_types.go

```diff
@@ -302,7 +302,7 @@ type GlanceSection struct {
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	// APIOverride, provides the ability to override the generated manifest of several child resources.
-	APIOverride Override `json:"apiOverride,omitempty"`
+	APIOverride map[string]Override `json:"apiOverrides,omitempty"`
```

Comment:
I think the samples below need to be updated to reflect this change.  i.e. they still have:

```
  glance:
    apiOverride:
      route: {}
```

Reply:
Yeah, updating the related sample files in my next PS.